test/e2e_node/jenkins/e2e-node-jenkins.sh

```diff
@@ -28,7 +28,16 @@ set -x
 
 . $1
 
+if [ "$INSTALL_GODEP" = true ] ; then
```

Comment:
i think i took the need for this out a while ago when e2e-node was updated, i left the `if` in here just in case it was needed again.  should i delete entirely ?